go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/niftynei/glightning
+
+go 1.16
+
+require github.com/stretchr/testify v1.7.0
```

Comment:
From CD/CD I found the following error

```
# github.com/stretchr/testify/assert
../../stretchr/testify/assert/assertions.go:1738:5: undefined: errors.Is
../../stretchr/testify/assert/assertions.go:1761:6: undefined: errors.Is
../../stretchr/testify/assert/assertions.go:1784:5: undefined: errors.As
../../stretchr/testify/assert/assertions.go:1801:7: undefined: errors.Unwrap
../../stretchr/testify/assert/assertions.go:1805:7: undefined: errors.Unwrap
?   	github.com/niftynei/glightning/examples/plugin/pl_btc	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_dbwrites	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_example	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_featurebits	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_htlcacc	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_method	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_openchan	[no test files]
?   	github.com/niftynei/glightning/examples/plugin/pl_rpccmd	[no test files]
?   	github.com/niftynei/glightning/examples/rpc	[no test files]
?   	github.com/niftynei/glightning/gbitcoin	[no test files]
FAIL	github.com/niftynei/glightning/glightning [build failed]
FAIL	github.com/niftynei/glightning/glightning/tests [build failed]
FAIL	github.com/niftynei/glightning/jrpc2 [build failed]

```

Maybe we need to downgrade some packages or update the test?